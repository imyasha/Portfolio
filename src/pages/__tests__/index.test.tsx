import React from 'react';
import * as Gatsby from 'gatsby';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import IndexPage from '../index';

const useStaticQuery = jest.spyOn(Gatsby, 'useStaticQuery');
useStaticQuery.mockImplementation(() => ({
  site: {
    siteMetadata: {
      description: '',
      title: 'Home page',
      lang: 'en',
      meta: [],
      keywords: [],
    },
  },
}));

test.skip('renders correctly', () => {
  const { container } = render(<IndexPage />);
  expect(container.firstChild).toMatchSnapshot();
});
