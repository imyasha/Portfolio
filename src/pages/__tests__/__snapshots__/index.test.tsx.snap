// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: absolute;
  top: 0;
  left: 0;
  z-index: -1;
  width: 100vw;
  height: 100vh;
}

@media screen and (max-width:768px) {

}

<div
  class="c0"
  id="tsparticles"
>
  <canvas
    class="tsparticles-canvas-el"
    height="0"
    style="width: 100%; height: 100%;"
    width="0"
  />
</div>
`;
